jbmc/src/java_bytecode/select_pointer_type.h

```diff
@@ -39,8 +39,7 @@ class select_pointer_typet
   virtual pointer_typet convert_pointer_type(
     const pointer_typet &pointer_type,
     const generic_parameter_specialization_mapt
-      &generic_parameter_specialization_map,
-    const namespacet &ns) const;
```

Comment:
Parameter is required in derived classes that override this method.

Reply:
Parameter restored and documented.